static/app/components/demo/demoHeader.tsx

```diff
@@ -116,14 +120,34 @@ const BaseButton = styled(Button)`
   text-transform: uppercase;
 `;
 
+const RequestDemoBtn = styled(BaseButton)`
+  @media (max-width: ${p => p.theme.breakpoints[0]}) {
+    display: none;
+  }
+`;
+
 // Note many of the colors don't come from the theme as they come from the marketing site
 const GetStarted = styled(BaseButton)`
   border-color: transparent;
   box-shadow: 0 2px 0 rgb(54 45 89 / 10%);
   background-color: #e1567c;
   color: #fff;
+  .short-text {
+    display: none;
+  }
+  @media (max-width: 650px) {
+    .long-text {
+      display: none;
+    }
+    .short-text {
```

Comment:
@Neo-Zhixing Nit: I think could make styled components of the `span` instances and then write your styled component with a selector with the styled component itself with string interpolation. Or maybe I'm wrong lol.